test/tests/userobjects/nearest_point_layered_average/tests

```diff
@@ -20,10 +20,23 @@
     prereq = test
   []
 
+  [from_positions]
+    type = 'Exodiff'
+    input = 'nearest_point_layered_average.i'
+    cli_args = "Positions/file/type=FilePositions
+                Positions/file/files=points.txt
+                UserObjects/npla/positions_object=file"
+    exodiff = 'nearest_point_layered_average_out.e'
```

Comment:
this is going to fail recover you need to prereq from_file as they use the same gold file

Reply:
Changed as suggested. Passed on my local machine.